service/handlers.go

```diff
@@ -201,3 +201,16 @@ func (rh *RouteHandler) convertToWebhookGenericObject(genericObject client.Gener
 		Config: config,
 	}, nil
 }
+
+func (rh *RouteHandler) isUniqueName(webhookName string, projectID string, apiClient client.RancherClient) (int, error) {
+	objs, err := apiClient.GenericObject.List(&client.ListOpts{})
```

Comment:
Can you filter by name here so that you don't have to loop through all objects? If that filter doesn't work on the cattle side, we'll have to add it. Looping through all isn't really acceptable.